rules-reviewed/jboss-eap/jboss-eap5/jboss-eap5-xml.windup.xml

```diff
@@ -0,0 +1,338 @@
+                    </description>
+                    <link href="https://docs.jboss.org/seam/latest/reference/html_single/" title="Seam Framework Reference"/>
+                    <tag>seam</tag>
+                </classification>
+            </perform>
+        </rule>
+        <rule id="jboss-eap5-xml-07000">
+            <when>
+                <xmlfile matches="/page" in="{*}.page.xml"/>
+            </when>
+            <perform>
+                <classification title="JBoss Seam Page" effort="3"/>
+            </perform>
+        </rule>
+
+
+        <rule id="jboss-eap5-xml-08000">
+            <when>
+                <xmlfile matches="//*[local-name()='jboss-app']" in="jboss-app.xml"/>
+            </when>
+            <perform>
+                <classification title="JBoss EAR Configuration" effort="1">
+                    <description>
+                        The `jboss-app.xml` file configures a Java EE application deployments specifically for JBoss EAP.
+                    </description>
+                    <!--<link href="" title=""> TODO: ESSC-25 -->
+                    <tag>deployment</tag>
+                    <tag>descriptor</tag>
+                </classification>
+            </perform>
+        </rule>
+        <rule id="jboss-eap5-xml-09000">
+            <when>
+                <xmlfile matches="//*[local-name()='jboss-web']" in="jboss-web.xml"/>
+            </when>
+            <perform>
+                <classification title="JBoss Web Application Descriptor" effort="1">
+                    <description>
+                        The `jboss-web.xml` file configures a Java EE web application specifically for JBoss EAP.
+                        It is an extension to standard `web.xml`.
+                    </description>
+                    <tag>web</tag>
+                    <tag>deployment</tag>
+                    <tag>descriptor</tag>
+                </classification>
+            </perform>
+        </rule>
+        <rule id="jboss-eap5-xml-10000">
+            <when>
+                <xmlfile matches="//*[local-name()='classloading']" in="jboss-classloading.xml"/>
+            </when>
+            <perform>
+                <classification title="JBoss 5 Classloader Configuration" effort="5">
+                    <description>
+                        The `jboss-classloading.xml` file allows customization of classloading in JBoss EAP 5.
+                    </description>
+                    <link title="JBoss EAP 5 Classloading documentation" href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/5/html/Microcontainer_User_Guide/sect-JBoss_Microcontainer_User_Guide-The_ClassLoading_Layer-ClassLoading.html"/>
+                    <tag>classloading</tag>
+                </classification>
+                <hint effort="5" severity="MANDATORY">
+                    <message>
+                        Classloading in EAP 6 is of modular nature, based on JBoss Modules.
```

Comment:
Though I think that in this case, a description of the problem would also be a good idea, such as "This configuration is not supported in EAP6, instead...."